cartographer/io/hybrid_grid_points_processor.h

```diff
@@ -0,0 +1,54 @@
+#ifndef CARTOGRAPHER_IO_HYBRID_GRID_POINTS_PROCESSOR_H_
+#define CARTOGRAPHER_IO_HYBRID_GRID_POINTS_PROCESSOR_H_
+
+// Library used for processing points processor pipeline.
+
+#include <memory>
+#include <string>
+
+#include "cartographer/io/file_writer.h"
+#include "cartographer/io/points_batch.h"
+#include "cartographer/io/points_processor.h"
+#include "cartographer/mapping_3d/hybrid_grid.h"
+#include "cartographer/mapping_3d/range_data_inserter.h"
+
+namespace cartographer {
+namespace io {
+
+// Creates a geolocated hybrid grid of the points with voxels being
```

Comment:
This is not in this PR, though. And Cartographer always works in a metric space, there is no geodetic pose in the system anywhere, so I do not think this is a good fit for this repository. Jie wanted to do something similar before and side-loaded the information in google3, I suggest the same here. 

Remove 'geolocated', since this is not geolocated.